### ISSUE TITLE
Incompatible issue

### ISSUE THREAD

**Issue by iamsubhaa** [ammarahm-ed/react-native-admob-native-ads#15]
Incompatible issue with Gradle 7.0

**Comment by ammarahm-ed** [comment]
@iamsubhaa which react native version are you using? the latest rn 0.62.2 uses gradle 6.0...

**Comment by iamsubhaa** [comment]
0.62.2 actually problem with mergemultidex problem

**Comment by ammarahm-ed** [comment]
In your project/android/app/build.gradle add this in dependancies:

```
implementation 'com.android.support:multidex:2.0.1'
```


and in defaultConfig set `multiDexEnabled` to true

```
 defaultConfig {
        multiDexEnabled true  
    }
```

**Comment by ammarahm-ed** [comment]
I am closing this due to inactivity

**Comment by iamsubhaa** [comment]
After add you have mentioned lines in build.gradle app force closed

**Comment by iamsubhaa** [comment]
> After add you have mentioned lines in build.gradle app force closed

Check about it

**Comment by ammarahm-ed** [comment]
@iamsubhaa The problem you are facing is not due to this library but something else is causing the issue. You are still facing the same problem?  Try the example App by cloning this project and report the issue there.

**Comment by ammarahm-ed** [comment]
@iamsubhaa update to the latest version of the library first.

**Comment by iamsubhaa** [comment]
Example app run awesome but myapp not work
React-native 0.62.0
Gradle 7.0

**Comment by ammarahm-ed** [comment]
@iamsubhaa I don't think the issue is related to this library.

**Comment by amin79** [comment]
@iamsubhaa I have same problem. Did you find any solution? As soon as I install the library, without doing anything, my App crashes.

Build is successful but I got the message: Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.

"react-native": "0.62.2",
classpath("com.android.tools.build:gradle:3.6.3")

I also added this line in build.gradle: implementation "com.android.support:multidex:2.0.1" and multiDexEnabled true.
